homeassistant/helpers/config_validation.py

```diff
@@ -885,6 +885,8 @@ def custom_serializer(schema):
 
 DEVICE_ACTION_SCHEMA = DEVICE_ACTION_BASE_SCHEMA.extend({}, extra=vol.ALLOW_EXTRA)
 
+_SCRIPT_SCENE_SCHEMA = vol.Schema({vol.Required("scene"): string})
```

Comment:
Let's validate it as entity ID for the scene domain.

```suggestion
_SCRIPT_SCENE_SCHEMA = vol.Schema({vol.Required("scene"): entity_domain("scene")})
```